package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "hello_microverse",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

Comment:
[Optional] To make the process of fixing linter issues, consider adding the following scripts:
```
"css-lint": "npx stylelint \"**/*.{css,scss}\"",
"css-lint-fix": "npx stylelint \"**/*.{css,scss}\" --fix"
```
That will allow you to see lint issues by simple running the command `npm run css-lint` to see all lint issues in your CSS and using `npm run css-lint-fix` will automatically fix the easy to solve problems with your CSS